src/index.html

```diff
@@ -3,17 +3,29 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      href=" https://cdn.jsdelivr.net/npm/modern-normalize@1.1.0/modern-normalize.min.css "
```

Comment:
`href="https://cdn.jsdelivr.net/npm/modern-normalize@1.1.0/modern-normalize.min.css"` - bez spacji?

Reply:
Nie ma to większego znaczenia bo działa, ale poprawie w następnym commicie.